src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap

```diff
@@ -431,13 +431,23 @@ exports[`StoryPromo Container should render correctly for canonical 1`] = `
     <div
       className="c2"
     >
-      <img
-        alt="Image Alt text"
-        className="c3"
-        height={null}
-        src="https://ichef.bbci.co.uk/news/660/cpsprodpb/0A06/production/image.jpg"
-        width={2048}
+      <div
+        className="lazyload-placeholder"
+        style={
+          Object {
+            "height": undefined,
```

Comment:
Not quite sure whats going on here, but its the same on latest so /shrug https://github.com/bbc/simorgh/blob/6baced13e2bb8a46ac6eb9884f407a1ea817c50b/src/app/containers/Figure/__snapshots__/index.test.jsx.snap#L34-L41